codegens/nodejs-unirest/test/unit/snippet.test.js

```diff
@@ -48,6 +48,24 @@ describe('nodejs unirest convert function', function () {
       });
     });
 
+    it('should return snippet with ES6 features', function () {
+      request = new sdk.Request(mainCollection.item[0].request);
+      options = {
+        ES6_enabled: true
+      };
+      convert(request, options, function (error, snippet) {
+        if (error) {
+          expect.fail(null, null, error);
+          return;
+        }
+        expect(snippet).to.be.a('string');
+        snippetArray = snippet.split('\n');
+        expect(snippetArray[0]).to.equal('const unirest = require(\'unirest\');');
+        expect(snippetArray).to.include(`const req = unirest('${mainCollection.item[0].request.method}', ` +
+        `'${mainCollection.item[0].request.url.raw}')`);
```

Comment:
Can you also add an expect for `.end` arrow function?

Reply:
@shreys7 Actually, initially I tried for that but don't know somehow I was getting error maybe because of some extra space. 
Let me try again.

Reply:
@shreys7 I have updated, please have a look.